app/src/main/java/com/example/android/weatherapp/utilities/Helper.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.android.weatherapp.utilities
+
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.*
+import kotlin.math.round
+
+private const val DATE_FORMAT = "MMMM dd, yyyy"
+
+object Helper {
+    const val APP_ID = "4dfdfc7144138f43bfa36b5b3a4b097f"
+
+    fun getDayFromDate(inputDate: String): String{
+        val format: DateFormat = SimpleDateFormat("yyyy-MM-dd hh:mm:ss", Locale.ENGLISH)
```

Comment:
`"yyyy-MM-dd hh:mm:ss"` this can be moved to strings ?

Reply:
Moved the string to constant file